src/connectors/aws_account.py

```diff
@@ -0,0 +1,100 @@
+"""AWS Account List
+Lists your AWS Accounts
```

Comment:
We say "Collects" on every other one, so let's do that here, and clarify that it's "in your Organization"?

Reply:
resolved